components/Header/index.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledHeader = styled.header`
+  position: sticky;
+  top: 0;
+  background-color: #fada5e;
+  padding: 3px;
+  text-align: center;
```

Comment:
I'd suggest working with flexbox instead of `text-align`. As soon as you introduce more content to your header, `text-align` won't suffice for layouting purposes.